src/store/account.ts

```diff
@@ -54,13 +54,14 @@ interface AccountState {
   localUsernames: string[];
   lastUsername?: string;
   signupAccount?: SignupAccount;
-  loginAccount?: LoginAccount;
+  loginAccount: LoginAccount | {};
```

Comment:
Didn't get this change

Reply:
What will mean loginAccount will be truth in any case, that's not we expect

Reply:
that was only for initial state

Reply:
okay, I reverted it